gemrb/GUIScripts/GUISTORE.py

```diff
@@ -121,12 +118,9 @@ def CloseStoreWindow ():
 
 	GemRB.LeaveStore ()
 	
-	if GemRB.GetVar ("Inventory"):
-		GemRB.SetVar ("Inventory", 0)
-		GUIINV.OpenInventoryWindow ()
-	else:
+	if GemRB.GetView("WIN_INV") == None:
 		GemRB.GamePause (0, 3)
-		GUICommonWindows.SetSelectionChangeHandler( None )
+	else:
 		GUICommonWindows.CloseTopWindow()
```

Comment:
isn't bag use broken now, since both windows are top windows, so the inventory got closed on store entry and we're not reopening it? So even the view fetch itself would fail here?
Also, since the store window is really two windows, doesn't the other one still need closing?